lib/tasks/translation.rake

```diff
@@ -87,21 +88,16 @@ namespace :translation do
   desc "Add missing translations"
   task(:add_missing, [:locale] => [:environment]) do |t, args|
     locale = args[:locale]
-    require 'i18n/tasks/cli'
 
-    i18n_args = ["add-missing", "--nil-value"]
-    i18n_args << ["-l", locale] if locale
-
-    I18n::Tasks::CLI.start(i18n_args.flatten)
+    I18n::Tasks::CLI.start(TranslationHelper.new(["add-missing", "--nil-value"], locale).with_optional_locale)
```

Comment:
You can now pass `args[:locale]` directly, and remove the `locale=` variable on line 90, as it only gets used here. Same comment for the other rake task.

Reply:
@ChrisBAshton good point, it is fixed!